docs/tables/github_audit_log.md

```diff
@@ -0,0 +1,60 @@
+# Table: github_audit_log
+
+The `github_audit_log` table helps to find all audit events for an organization.
+
```

Comment:
From my understanding, this table only works for organizations that have GitHub Enterprise enabled. If that's the case, can you please add a note in this table doc mentioning that requirement?